src/components/People.js

```diff
@@ -87,77 +81,81 @@ class People extends React.Component {
       return dataList;
   }
 
-  saveReady = () => {
-      this.setState({noSave: false});
+  const saveReady = () => {
+      setNoSave(false);
   }
 
-  alertTimer = () => {
-      setTimeout(function () {
-          this.setState({justSaved: false});
-      }.bind(this), 3000);
+  const alertTimer = () => {
+      setTimeout(() => {
+          setJustSaved(false);
+      }, 3000);
   }
 
-  render() {
-      let leaving = this.generateRows(this.props.people.leaving);
-      let staying = this.generateRows(this.props.people.staying);
-      let faculty = this.generateRows(this.props.people.faculty);
-
-      let maybeHeader  = '';
-      let maybeHR = '';
-      if (staying.length > 0) {
-          maybeHeader = (<h3>In-Batch</h3>);
-          maybeHR = (<hr />);
-      }
-      if (this.state.justSaved) {
-          this.alertTimer();
-      }
+  let leaving = generateRows(props.people.leaving);
+  let staying = generateRows(props.people.staying);
+  let faculty = generateRows(props.people.faculty);
 
-      const staffRows = faculty.map((row) => (
-          <PeopleRow fromMe={this.props.fromMe} data={row} saveReady={this.saveReady} updated_niceties={this.state.updated_niceties}/>
-        ))
-
-      return (
-          <div className="people">
-            <Modal show={this.state.justSaved}>
-              <Modal.Body>
-                Niceties Saved!
-              </Modal.Body>
-            </Modal>
-            <div className="save_button">
-              <SaveButton
-                noSave={this.state.noSave}
-                onClick={this.saveAllComments}>
-                Save
-              </SaveButton>
-            </div>
-            <Grid>
-              <hr />
-              <h3>Leaving Soon</h3>
-              {leaving.map((row) => (
-                  <PeopleRow fromMe={this.props.fromMe} data={row} saveReady={this.saveReady} updated_niceties={this.state.updated_niceties}/>
-                ))}
-
-              <hr />
-              { maybeHeader }
-              {staying.map((row) => (
-                  <PeopleRow fromMe={this.props.fromMe} data={row} saveReady={this.saveReady} updated_niceties={this.state.updated_niceties}/>
-                ))}
-
-              { maybeHR }
-              <h3>Staff</h3>
-              { staffRows }
-            </Grid>
-            <div className="save_button">
-              <SaveButton
-                noSave={this.state.noSave}
-                onClick={this.saveAllComments}>
-                Save
-              </SaveButton>
-            </div>
-          </div>
-      );
-    }
+  let maybeHeader  = '';
+  let maybeHR = '';
+  if (staying.length > 0) {
+      maybeHeader = (<h3>In-Batch</h3>);
+      maybeHR = (<hr />);
+  }
+  if (justSaved) {
+      alertTimer();
+  }
 
+  const leavingRows = leaving.map((row) => (
```

Comment:
I thought it was weird that staffRows was pulled out into its own variable, so I made parallel leavingRows and stayingRows.